fenix/app/src/main/res/layout/full_screen_notification_dialog.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Reply:
Changed here and in dialog_full_screen_notification.xml 👍 

❓This is how the default formatter handles these comments in xml - is there a project formatter that enforces this? I notice we have 30+ other instances of this formatting in xml where the comment is wrapped from the top of the file

Reply:
There isn't a project formatter, but it would probably make sense to add a linter for this as a followup bug.

Reply:
Created https://bugzilla.mozilla.org/show_bug.cgi?id=1863755